code/obj/artifacts/artifact_items/pitcher.dm

```diff
@@ -0,0 +1,192 @@
+/obj/item/reagent_containers/food/drinks/drinkingglass/artifact
+	name = "artifact pitcher"
+	icon = 'icons/obj/artifacts/artifactsitem.dmi'
+	desc = "You have no idea what this thing is!"
+	artifact = 1
+	module_research_no_diminish = 1
+	mat_changename = 0
+	can_recycle = 0
+
+	New(var/loc, var/forceartitype)
+		..()
+		var/datum/artifact/pitcher/AS = new /datum/artifact/pitcher(src)
+		if (forceartitype)
+			AS.validtypes = list("[forceartitype]")
+		src.artifact = AS
+		SPAWN_DBG(0)
+			src.ArtifactSetup()
+
+		gulp_size = rand(2, 10) * 5 //How fast will you drink from this? Who knows!
+		var/capacity = rand(5,20)
+		capacity *= 100
+		src.reagents.maximum_volume = capacity
+		//Fun stuff
+		if (prob(7))
+			reagents.add_reagent("dbreath", 30)
+		if (prob(7))
+			reagents.add_reagent("freeze", 30)
+		if (prob(3))
+			reagents.add_reagent("stimulants", 20)
+		if(prob(15))
+			reagents.add_reagent("hairgrownium", 30)
+		if (prob(10))
+			reagents.add_reagent("super_hairgrownium", 20)
+		if(prob(7))
+			reagents.add_reagent("reversium", 15)
+		if (prob(12))
+			reagents.add_reagent("strange_reagent", 20)
+		if (prob(10))
+			reagents.add_reagent("booster_enzyme", 30)
+		if (prob(10))
+			reagents.add_reagent("hugs", 25)
+		if (prob(10))
+			reagents.add_reagent("love", 25)
+		if (prob(10))
+			reagents.add_reagent("colors", 40)
+		if (prob(7))
+			reagents.add_reagent("fliptonium", 50)
+		if (prob(3))
+			reagents.add_reagent("glowing_fliptonium", 3)
+		if (prob(10))
+			reagents.add_reagent("fartonium", 30)
+		if (prob(10))
+			reagents.add_reagent("glitter", 30)
+		if (prob(10))
+			reagents.add_reagent("voltagen", 50)
+		if (prob(5))
+			reagents.add_reagent("rainbow fluid", 30)
+		if (prob(1))
+			reagents.add_reagent("vampire_serum", 5)
+		if (prob(3))
+			reagents.add_reagent("painbow fluid", 10)
+		if (prob(1))
+			reagents.add_reagent("werewolf_serum", 2)
+		/* Removed pending patho rework, in case this becomes functional again
```

Comment:
Liquid plasma is a disease reagent, doesn't really have anything to do with pathology. :)
But removing it is probably a good idea, seeing as the disease doesn't do anything.

Reply:
Ah, thanks for the clarification. Fully removed now.